website/docs/using-unleash/deploy/license-keys.mdx

```diff
@@ -1,119 +1,83 @@
--   For **Unleash Open Source**, no license is necessary and these steps do not apply.
+**Plan**: [Enterprise](https://www.getunleash.io/pricing) | **Version**: `5.8+`
 
 :::
 
-Areas of license key management that will be covered include:
-
--   [License policy](#license-policy)
-    -   [Enforcement](#enforcement)
--   [Obtaining a new or updated license](#obtaining-a-new-or-updated-license)
-    -   [New Customers](#new-customers)
-    -   [Existing Customers](#existing-customers)
--   [Installing a new or replacing an existing license](#installing-a-new-or-replacing-an-existing-license)
-    -   [Admin UI](#admin-ui)
-    -   [Environment Variable](#environment-variable)
-
----
+Unleash Enterprise uses license keys to ensure you comply with the terms of your contract. For hosted versions of Unleash, license management is handled automatically. If you self-host Unleash Enterprise, you must manage your license key.
 
 ## License policy
 
-License keys are leveraged for self-hosted Unleash Enterprise instance(s) to be compliant with the following aspects of a customer agreement:
-
--   **Seat count** (# of licensed users)
--   **Contract duration** (expiration date)
+License keys check compliance with the following aspects of the customer agreement:
+-   **Seat count**: the number of licensed users
+-   **Contract duration**: whether the license is within the contract expiration date
 
-These parameters are encoded into the license key. Therefore, installation of a new license key per self-hosted instance will be required should changes to any of the above occur (for example, increased seat count or contract renewal).
+The seat count and contract duration parameters are encoded into the license key. Therefore, if you change your seat count or renew your contract, you must install a new license.
 
-Separate license keys are required per Unleash instance, for example, a dev or production instance would have separate keys.
+Each Unleash instance requires a separate license key. For example, a development or production instance must have separate keys.
 
 :::warning
-Reusing Unleash keys across multiple installations is prohibited under the terms of your license.
+Reusing Unleash license keys across multiple installations is prohibited under the terms of your license.
 :::
 
-### Enforcement
-
-The Unleash UI will display banners in any of the following non-compliant scenarios:
+### License enforcement
 
--   **No installed/missing license key**  
-    ![Banner showing a missing license](/img/license-banner-missing.png)
--   **Expired license key**  
-    ![Banner showing an expired license](/img/license-banner-expired.png)
--   **Enroled user count higher than the number of seats supported by the license**  
-    ![Banner showing a high user count](/img/license-banner-highuser.png)
-
----
+:::note Availability
 
-## Obtaining a new or updated license
+**Plan**: [Enterprise](https://www.getunleash.io/pricing) self-hosted | **Version**: `6.7+`
```

Comment:
What do we mean when we say that hosted Enterprise 'license management is handled automatically'. Does it mean that the following enforcement never applies to them?

Reply:
Handled automatically is probably not accurate as there's no license for hosted Enterprise. We just use different methods to validate that users stay within their contracted limits

Reply:
Ok, thanks! But do we have the same restrictions in place (read-only mode etc) in case they do not stay within the limits?

Reply:
We don't go into read-only mode, no. We handle things differently for hosted customers, mainly reaching out by email or customer success team.